src/next.md

```diff
@@ -7,6 +7,185 @@
+~~~
+
+#### Running ProblemListed tests
+
+To run ad-hoc runs of ProblemListed tests use `RUN_PROBLEM_LISTS=true`.
+
+~~~
+make test TEST=... JTREG=RUN_PROBLEM_LISTS=true
+~~~
+
+### Exclude jtreg tests using `@ignore`
+
+The `@ignore` keyword is used in the test source code. This is mainly used for tests that are so broken that they may be harmful or useless. Examples can be tests that don't compile because something changed in the platform; or a test which might remove your `/etc/shadow`. Use `@ignore` with a bug reference in the test case to prevent the test from being run.
+
+~~~java
+/**
+ *  @test
+ *  @ignore 4711
+ *
+~~~
+
+In this example, `MyTest.java` is excluded, tracked by bug `JDK-4711`. `@ignore` should always be placed directly before the first `@run` line in the test.
+
+### Dealing with JBS bugs for test exclusion
+
+ProblemListing and `@ignore`-ing are done in the JDK source tree, that means a check-in into the repository is needed. This in turn means that a unique JBS issue and a code review are needed. This is a good thing since it makes test problems visible.
+
+* **Code review**: This is considered a trivial change. It only needs a review from one official Reviewer and you don't need to wait 24 hours before commit, even in areas that usually have stricter rules around reviews.
+* **JBS issue**: A JBS issue is obviously created for the bug that caused the failure, we call this the _main issue_. To exclude the test, create a subtask to the main issue. Also add the label `problemlist` to the main issue.
+
+The fix for the main issue should remove the test from the ProblemList or remove the `@ignore` keyword from the test.
+
+#### Triage excluded issues
+
+After a failure is handled by excluding a test, the main JBS issue should be re-triaged and possibly given a new priority. This should be handled by the standard triage process. A test exclusion results in an outage in our testing. This outage should be taken into consideration when triaging, in addition to the impact of the bug itself.
+
+## Backing Out a Change
+
+If a change cause a regression that can't be fixed within reasonable time the best way to handle the regression can be to back out the change. Backing out means that the inverse (anti-delta) of the change is pushed to effectively undo the change in the repository. There are two parts to this task, how to do the bookkeeping in JBS, and how to do the actual backout in mercurial.
+
+The backout is a regular change and will have to go through the standard code review process, but is considered a trivial change and thus it requires only one Reviewer and will avoid the 24h code review window, even for areas where stricter rules usually applies. The rationale is that a backout is usually urgent in nature and the change itself is automatically created by hg, and reviewed by the person who is performing the backout, so even though only one additional Reviewer is required the change will in practice get two reviews.
```

Comment:
where is the 24h code review window discussed ?
It might be a sensible policy for non-urgent fixes but I've never seen it documented/codified and we've never had it as a rule in the client area.

Reply:
That text hasn't been written yet. The 24h-rule is in use in the VM. I don't know if any other area use this rule. It doesn't apply to urgent fixes. Once we write that text it should specify if there are differences between areas. Whether it's needed or not depends, I guess, on how distributed the active area developers are over timezones.

Reply:
given 24h rule is VM-specific, I think it might be reasonable to omit in this text and just say that backouts are considered trivial changes, and have the difference in rules for trivial and non-trivial changes described in yet-not-written reviewing doc (or its VM part)

Reply:
Yes, I agree. There is no need to define (again) what trivial means here. The word should be a link to where "trivial" is defined.